packages/video_player/video_player/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
-## NEXT
+## 2.3.1
```

Comment:
Sorry, I missed this before: this version change should be 2.4.0 rather than 2.3.1, since this is a new feature rather than a bugfix.

Reply:
Ok, I updated